lxd/storage/drivers/driver_lvm.go

```diff
@@ -448,14 +448,12 @@ func (d *lvm) Validate(config map[string]string) error {
 		"lvm.thinpool_name":          validate.IsAny,
 		"lvm.thinpool_metadata_size": validate.Optional(validate.IsSize),
 		"lvm.use_thinpool":           validate.Optional(validate.IsBool),
+		"lvm.vg.force_reuse":         validate.Optional(validate.IsBool),
 		"volume.block.mount_options": validate.IsAny,
```

Comment:
Why are "volume.block.mount_options" and "volume.block.filesystem" not handled by LVM's commonVolumeRules?

Reply:
I've moved "volume.block.filesystem" to `commonVolumeRules` but I'm not sure about `volume.block.mount_options` config. In current codebase this one isn't defined in the rules in `ValidateVolume` for lvm. It's only defined on pool level. What do you think about this?

Reply:
Done.